src/transform/src/union_cancel.rs

```diff
@@ -84,6 +101,32 @@ impl UnionBranchCancellation {
         Ok(())
     }
 
+    /// Returns the sign of a given union branch. The sign is -1 if the branch contains
```

Comment:
This might be totally dumb but how do we plan `repeat(-1)`? Does that involve a negate?

Reply:
`repeat` as in the string function? I don't think that involves a negate. Or is there any other `repeat` function?

Reply:
Oh, there is `repeat_row`, which is implemented as a table function:

```
materialize=> explain select * from t1, repeat_row(-1);
          Optimized Plan          
----------------------------------
 %0 =                            +
 | Get materialize.public.t1 (u1)+
 | FlatMap repeat_row(-1)        +
 | | demand = (#0)               +
 
(1 row)

materialize=>
```